WMF Framework/NavigationBar.swift

```diff
@@ -125,6 +125,17 @@ public class NavigationBar: SetupView, FakeProgressReceiving, FakeProgressDelega
             titleBarItems.append(rightBarButtonItem)
         }
         titleBar.setItems(titleBarItems, animated: false)
+        
+        
+        // HAX: workaround for iOS 13 constraint breakage warnings
+        if #available(iOS 13, *) {
+            guard let contentView = titleBar.subviews.first, String(describing: type(of: contentView)) == "_UIToolbarContentView" else {
+                return
+            }
+            contentView.constraints
+                .filter { $0.priority == .required && $0.firstAttribute == .leading || $0.firstAttribute == .trailing }
+                .forEach { $0.priority = .defaultHigh }
```

Comment:
The end result at runtime is likely the same since the warning is about it breaking the constraint. This just makes that constraint breakage OK by lowering the priority. Given that it's only silencing the warning, I don't think this is worth it. It's only really worth fixing if there's a way to alter the setup we can control without having to dig into the internals of `UIToolbar`

Reply:
The constraints don't break with this because of the way the priorities are compared, but I hear ya. It's not a good fix.

Reply:
I'm defining a constraint 'break' as a constraint not being fully satisfied - the priorities are there to determine which constraint that can stay unsatisfied (or 'broken') when there's a conflict

Reply:
Ah, I see. I was thinking more literally. One of the things I like about SwiftUI is, iirc, it won't produce output with any unsatisfiable constraints, so in some distant future we won't be using  non-`.required` priorities (or eventually any😜 manual constraints) all over the place, as we do now.

Reply:
I for one welcome our reactive declarative one-way data flow overlords 😂